frontend/helpers/path-helpers.ts

```diff
@@ -3,14 +3,14 @@ import { invariant } from '@ifixit/helpers';
 import {
    ProductList,
    ProductListType,
+   StorePage,
    iFixitPageType,
-   iFixitPage,
 } from '@models/product-list';
 import { GetServerSidePropsContext } from 'next';
 import { stylizeDeviceTitle } from './product-list-helpers';
 
 type ProductListPathAttributes = Pick<
-   ProductList | iFixitPage,
+   ProductList | StorePage,
```

Comment:
I wasn't aware of that, thanks for cluing me in. Since the intent of this PR is to set up zod schemas for product list caching, let's do the store page breadcrumb refactor in a separate PR.